pages/upgrading_an_app.md

```diff
@@ -70,7 +70,7 @@ Here is how the upgrade process works graphically (read the sections below to ha
 
 ![GitGraph]({{ site.url }}/images/upgrade_gitgraph.png)
 
-(this image comes from [JSFiddle](http://jsfiddle.net/lordlothar99/tqp9gyu3/) )
+(this image comes from Mermaid [GitHub Gist](https://gist.github.com/timothystone/aa5bcd27d1ede5a08a7065e2e1a1cb77) )
```

Comment:
Can you get a high-res version of this image and update the `upgrade_gitgraph.png` file? It looks a little grainy in the preview: https://deploy-preview-1326--jhipster-site.netlify.app/upgrading-an-application/